CHANGELOG.rst

```diff
@@ -2,6 +2,18 @@
 CHANGELOG
 =========
 
+2.2.1 (unreleased)
```

Comment:
```suggestion
2.3.0 (unreleased)
```

This should be `2.3.0` given it introduces new features.

Reply:
Fair, but given we introduced cmdline option and a new env var, we should consider it a feature release. 👍